modules/transactions/src/main/java/org/apache/ignite/internal/tx/impl/TxMessageSender.java

```diff
@@ -66,6 +68,9 @@ public class TxMessageSender {
 
     private final TransactionConfiguration transactionConfiguration;
 
+    // TODO https://issues.apache.org/jira/browse/IGNITE-22522 Remove it.
+    private final boolean enabledColocationFeature = getBoolean(COLOCATION_FEATURE_FLAG, false);
```

Comment:
We write this spell in many places. I suggest to introduce a static boolean field `ENABLED_COLOCATION_DEFAULT` with value `false` to make it easier to switch it globally and use it in all instances of this spell

Reply:
ENABLED_COLOCATION_DEFAULT added, usages adjsuted.